### ISSUE TITLE
"Get latest Translations" fails(?) without error

### ISSUE THREAD

**Issue by Kunsi** [zammad/zammad#502]
Hello,

i'm trying to update my Translations, however, clicking the Button in web interface results in a "please wait"-window, but then doesn't go any further. Window does never disappear even after one hour. Also there's no "close" button, so only way to get rid of it is a page refresh.

/var/log/zammad/production.log tells me:
```
I, [2016-12-04T14:34:41.820321 #21500]  INFO -- : Started GET "/api/v1/translations/sync/en-us?_=1480856880788" for 2001:bc8:225e:1001::200 at 2016-12-04 14:34:41 +0100
I, [2016-12-04T14:34:41.822751 #21500]  INFO -- : Processing by TranslationsController#sync as JSON
I, [2016-12-04T14:34:41.822876 #21500]  INFO -- :   Parameters: {"_"=>"1480856880788", "locale"=>"en-us", "translation"=>{}}
I, [2016-12-04T14:34:44.847355 #21500]  INFO -- : Completed 200 OK in 25025ms (Views: 0.3ms | ActiveRecord: 2.9ms)
```
(No more stuff regarding web interface, only regular checking of IMAP folder)

Zammad version: 1.1.0-1480452968.7140be8.jessie (https://deb.packager.io/gh/zammad/zammad)
System: VM, Debian jessie, all Updates installed

**Comment by thorsteneckel** [comment]
Hi @Kunsi - Is this behavior reproducible for you?

**Comment by monotek** [comment]
@thorsteneckel 
I think @martini just did some changes which could be resposible for the error: https://github.com/zammad/zammad/commit/5856dd7c06bc836285ac5700e37a923087a4f9a6
He's already working on a fix as far as i know.

**Comment by martini** [comment]
@monotek the change is currently on develop only

@Kunsi I just verified it, it's working fine for me (right now). Can you check if it's still not working for you? And if it's not working, can you have a look into the browser javascript console. Any errors in there? Thanks!

**Comment by martini** [comment]
JFI: Now I was able to reproduce. I'm on it...

**Comment by Kunsi** [comment]
Just to note: Doesn't work for me right now, also no errors on Browser Javascript Console.

**Comment by martini** [comment]
Thanks for the report, fixed in develop, 1.0 and 1.1 https://github.com/zammad/zammad/commit/0c2be657be80e987e19ec547cdd384231886a08c.
